.circleci/config.yml

```diff
@@ -188,6 +188,12 @@ run_oss_amp: &run_oss_amp
        command: |
          python benchmarks/oss.py --amp --epochs 3 --optim_type oss_sharded_ddp
 
+run_oss_for_each: &run_oss_for_each
+   - run:
+       name: Run OSS with Torch AMP and ForEach optmizer
+       command: |
+         python benchmarks/oss.py --amp --epochs 3 --optim_type oss_sharded_ddp --multi_tensor_optim
```

Comment:
Why do we want to add a regression benchmark? Would tests suffice?

Reply:
in that case it's not a regression, it just checks that it runs ?

Reply:
Replied below: You can do a golden weight comparison check which makes sure that the results match with the baseline optimizer (if that is what you want)

Reply:
no so there are two things in this PR, not completely related:
- restore the regression test for ShardedDDP which was broken
- start a test job with these new optimizers. Right now they are not 1:1 correct, it's being changed as I write so it should soon be the case, I thought that having a test job on the topic could be useful to check parity and catch early any eventual issues